Localization/pl-PL.hjson

```diff
@@ -2,154 +2,154 @@ Mods: {
 				}
 
 				OnEnter: {
-					// Label: Activate On Enter
-					// Tooltip: Toggle if magnet should be automatically activated when entering the world
+					Label: Włącz przy wejściu do świata
+					Tooltip: Przełącz, jeśli magnes ma być automatycznie włączany przy wejściu do świata.
 				}
 
 				NeedsSpace: {
-					// Label: Needs Space
-					// Tooltip: Toggle if magnet should only work when there is space in the inventory
+					Label: Wymaga miejsca w ekwipunku
+					Tooltip: Przełącz, jeśli magnes ma działać tylko jeśli jest wolne miejsce w ekwipunku.
 				}
 
 				Instant: {
-					// Label: Instant
-					// Tooltip: Toggle if items should teleport instead of home - This ignores any velocity/acceleration, but not range
+					Label: Natychmiastowy magnes
+					Tooltip: Przełącz, jeśli przedmioty mają być teleportowane do gracza, zamiast być przyciągane - ignoruje to jakąkolwiek prędkość/przyspieszenie, ale nie zasięg
 				}
 
 				Range: {
-					// Label: Range
-					// Tooltip: Base magnet radius in tiles
+					Label: Zasięg
+					Tooltip: Podstawowy promień działania magnesu, wyrażony w blokach.
 				}
 
 				Velocity: {
-					// Label: Velocity
-					// Tooltip: Speed at which items get pulled towards you
+					Label: Prędkość
+					Tooltip: Prędkość z jaką przedmioty są przyciągane do gracza.
 				}
 
 				Acceleration: {
-					// Label: Acceleration
-					// Tooltip: How fast an item reaches its peak speed
+					Label: Przyspieszenie
+					Tooltip: Jak szybko przedmiot osiąga maksymalną prędkość.
 				}
 
 				ScaleMode: {
-					// Label: Scale
-					// Tooltip: Scaling Mode
+					Label: Skalowanie
+					Tooltip: Rodzaj skalowania
 				}
 
 				CurrentMaxRange: {
-					// Label: Resulting Max Range
-					// Tooltip: ""
+					Label: Maksymalny zasięg
+					Tooltip: ""
 				}
 
 				CurrentVelocity: {
-					// Label: Resulting Velocity
-					// Tooltip: ""
+					Label: Prędkość
+					Tooltip: ""
 				}
 
 				CurrentAcceleration: {
-					// Label: Resulting Acceleration
-					// Tooltip: ""
+					Label: Przyspieszenie
+					Tooltip: ""
 				}
 			}
 
 			FilterListModeType: {
-				// Tooltip: ""
-				// Blacklist.Label: Blacklist
-				// Whitelist.Label: Whitelist
+				Tooltip: ""
+				Blacklist.Label: Czarna lista
+				Whitelist.Label: Biała lista
 			}
 
 			ScaleModeType: {
-				// Tooltip: ""
-				// Bosses.Label: Increase With Bosses
-				// AlwaysMaxRange.Label: Bosses + Max Range
-				// OnlyConfig.Label: Custom + Max Range
+				Tooltip: ""
+				Bosses.Label: Zwiększaj wraz z pokonanymi bossami
+				AlwaysMaxRange.Label: Bossowie + Maksymalny zasięg
+				OnlyConfig.Label: Własne + Maksymalny zasięg
 			}
 
-			// Common.AcceptClientChanges: Only the host of this world can change the config! Do so in singleplayer.
-		}
+			Common.AcceptClientChanges: Tylko gospodarz tego świata może zmieniać ustawienia konfiguracji! Zrób to w trybie jednoosobowym.
+		} 
```

Comment:
unnecessary space

Reply:
Not sure what do you mean, I cant see anything here

Reply:
Done